### PR TITLE
session_0

### PR DESCRIPTION
**Pull request by sravanireddy1102**
done with session 0 and 1 but I am unable to upload hello_world project ,,its been a day since I am trying to do that.
@abhishekkumar2718  can you please help me

### CODE REVIEW

**Comment by addhyanmalhotra** [comment]
Hi,
its seems like you have not executed the command
> cp page_controller_test.rb hello_world/test/controllers

once you have done this simply add this to your vcs

> git add hello_world/test/controllers/page_controller_test.rb

followed by commit and push.

Also can you push session_1 to a seperate branch.
>git checkout -b sravanireddy_session_1
>git add session_1/*

File: session_0/word_processing.rb [hunk 1]
```diff
@@ -18,7 +18,8 @@ def lower_case!(words)
 # words_with_prefix('apple', ['apple', 'ball', 'applesauce']) would
 # return the words 'apple' and 'applesauce'.
 def words_with_prefix(prefix, words)
-  raise NotImplementedError # TODO
+  return  words.filter_map{|word| word if word.index(prefix)==0}
```
Comment (addhyanmalhotra) [inline session_0/word_processing.rb:21]: try using, select as well as start_with? methods

**Review by addhyanmalhotra — CHANGES_REQUESTED** [review]

**Comment by abhishekkumar2718** [comment]
Hey, you don't need to push session 1's code for this pull request. Don't mind the failing checks.

The easiest way to upload them would be:

```bash
git checkout -b session_1_sravani
git add session_1/hello_world
git commit -m 'Solution for Session 1'
git push --set-upstream origin session_1_sravani
```

**Comment by sravanireddy1102** [comment]
error: The following untracked working tree files would be overwritten by checkout:
        Desktop/IRIS-RoR-Bootcamp-2020-main/session_1/.gitignore
        Desktop/IRIS-RoR-Bootcamp-2020-main/session_1/README.md
        Desktop/IRIS-RoR-Bootcamp-2020-main/session_1/page_controller_test.rb
Please move or remove them before you switch branches.
Aborting
@abhishekkumar2718  This is the error im getting,eventhough I have removed them

**Comment by sravanireddy1102** [comment]
error: failed to push some refs to 'https://github.com/sravanireddy1102/IRIS-RoR-Bootcamp-2020'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.

**Comment by abhishekkumar2718** [comment]
Closed because of a more recent PR: #47

**Pull request closed without merging**